.github/workflows/live-test.yml

```diff
@@ -0,0 +1,53 @@
+name: Live tests for B2K
+on: [push]
```

Comment:
On all push or on pull requests?

Reply:
@sabbour Just changed them :), it was from my fork which got missed. thank you.

Reply:
@sabbour on all pull requests. :)